docs/Content/SbieCtrl_HideMessage.md

```diff
@@ -0,0 +1,18 @@
+# SbieCtrl_HideMessage 
+
+SbieCtrl_HideMessage is a user setting in [Sandboxie Ini](SandboxieIni.md). It specifies which of the SBIE messages should be hidden from popping up.
+
+```
+   .
+   .
+   .
+   [UserSettings_054A02CE]
+   SbieCtrl_HideMessage=1101
+   SbieCtrl_HideMessage=1102,Example Message
+```
+
+The first parameter is mandatory, specifies the id of the SBIE Messages to be hidden.
+
+The second parameter is optional, displayed as "Message Text" in UI. Notice: Even if this is configured, All messages of such will be hidden regardless of its text.
```

Comment:
```suggestion
The second parameter is optional, displayed as "Message Text" in UI.
```

I have doubts about the second sentence. Does it mean that there is no point in setting the second parameter?

Reply:
If you set it it will be shown in GUI, not if not set, see image attached:
![image](https://github.com/user-attachments/assets/d1ee5747-daf5-4888-a094-715eefd59fba)

Reply:
Yes the second parameter may be regarded useless considering [all messages of this id will be hidden](https://github.com/sandboxie-plus/sandboxie-docs/blob/63676e86af30e40d0af53642a9bff7454528f776/docs/Content/ConfigureMenu.md?plain=1#L77)

Reply:
Did some changes considering this situation.
Also checked the instance will be hidden if parameter is not set.

Reply:
![image](https://github.com/user-attachments/assets/269fd91d-26cb-42a4-8dc3-ab29b0090b8c)
So we maybe change the i18n info of sbie as well?